editor/package.json

```diff
@@ -14,37 +14,36 @@
     "clean": "rm -rf lib/",
     "check": "tsc --noEmit && pnpm run lint && pnpm run check-dependency-cruiser && pnpm run prettier-check && pnpm run test && pnpm run test-browser",
     "check-ci": "tsc --noEmit && pnpm run lint && pnpm run check-dependency-cruiser && pnpm run prettier-check && pnpm run test-ci && pnpm run test-browser",
-    "check-code": "tsc --noEmit && pnpm run lint && pnpm run check-dependency-cruiser && pnpm run prettier-check",
-    "lint": "pnpx eslint \"src/**/*.{js,ts,tsx}\" --quiet",
-    "lint-loud": "pnpx eslint \"src/**/*.{js,ts,tsx}\"",
+    "check-code": "tsc --version && tsc --noEmit && pnpm run lint && pnpm run check-dependency-cruiser && pnpm run prettier-check",
+    "lint": "eslint \"src/**/*.{js,ts,tsx}\" --quiet",
```

Comment:
`pnpx` had been changed [over a year ago](https://github.com/pnpm/pnpm/pull/4267) to become an alias for [`pnpm dlx`](https://pnpm.io/cli/dlx), which downloads and runs the latest version of a package, rather than using the locally installed version

Reply:
wow, good to know!